superset/views/datasource.py

```diff
@@ -55,6 +55,13 @@ def save(self):
         db.session.commit()
         return self.json_response(data)
 
+    @expose('/get/<datasource_type>/<datasource_id>/')
+    @has_access_api
+    def get(self, datasource_type, datasource_id):
```

Comment:
2 things:
- I would add a small if condition to make sure we are returning some data. 
- Can we get a small unit test here testing a valid/bad`datasource_type` and `datasource_id`